logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -232,6 +233,7 @@ actual class CallManagerImpl(
      * onCallingReady
      * Will start the handlers for: ParticipantsChanged, NetworkQuality, ClientsRequest and ActiveSpeaker
      */
+    @Suppress("LongMethod")
```

Comment:
It would be better maybe if we move each part to a separate function and call them inside onCallingReady ?

```
private fun onCallingReady() {
    initParticipantsHandler()
    initClientsHandler()
    initNetworkHandler()
}
```